Split/Engine/DefaultTreatmentManager.swift

```diff
@@ -199,7 +199,8 @@ class DefaultTreatmentManager: TreatmentManager {
     }
 
     private func isSdkReady() -> Bool {
```

Comment:
Just as a comment, we call this state `isSdkOperational` in JS SDK, to better distinguish the case where the sdk is ready from cache but not ready. If you think it is not necessary, skip the comment.

Reply:
Hey Doc, Thanks for the comment. We'll keep Ready From Cache to be aligned with Android.